### DISCUSSION TITLE
2006/08/30/%E3%80%8A%E8%81%8C%E5%9C%BA%E5%8A%A8%E7%89%A9%E8%BF%9B%E5%8C%96%E6%89%8B%E5%86%8C%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#111] [Announcements]
# 2006/08/30/%E3%80%8A%E8%81%8C%E5%9C%BA%E5%8A%A8%E7%89%A9%E8%BF%9B%E5%8C%96%E6%89%8B%E5%86%8C%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

先把本书的开篇问题拿来考考大家：

https://blog.qizhen.xyz/2006/08/30/%E3%80%8A%E8%81%8C%E5%9C%BA%E5%8A%A8%E7%89%A9%E8%BF%9B%E5%8C%96%E6%89%8B%E5%86%8C%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：


FU
2009年08月3日 14:23 编辑
你读这本书，跟我读这本书相差了3年。哈哈。不知给你留这个言你能否看见。这本书的确让我换了个角度看职场，感觉也理性多了。但……说不出来，但总觉得这本书写的还是有点不对劲。这书，我觉得可能是给我们这些狗狗提供了另外一种思维去理解一些事情，加入了许多公司的的许多普遍化现象。但我感觉这书写的还是有些片面……我不知写书的作者有着怎样的经历，他在社会上是什么样的层次，是否有过管理经验等等~而且它毕竟是本书，要是被它的思路套住了，去理解周围发生的事情，那么这种理解也是片面的。政治的东西，是讨厌而逃不掉的，去面对总比去逃避好，更重要的，也是对自己的锻炼。职场，是个锻炼的平台。

回复
Alen
2008年08月8日 09:13 编辑
通知-您的此篇日志已被收录 您好，我是微软（中国）Spaces精彩空间风云榜【职场●时尚】的评委。您的此篇日志已被收录，将在近期更新到Spaces精彩空间风云榜首页【职场●时尚】频道，地址：http://msnpopularspaces.spaces.live.com/ 
相关频道投稿请进投稿页面：http://msnpopularspaces.spaces.live.com/blog/cns!C8E555A687097EF8!140.entry感谢您对微软（中国）官方SPACES首页和Spaces精彩空间风云榜的支持。

回复
mi
2007年11月7日 17:34 编辑
无论出发点如何好~一但结果是不尽人意的，似乎一切行为都不再有意义~
“尤其是只是一味抱怨对个人进步没有任何好处。个人只有适应社会，适应周围的环境和制度，才能比较顺利的发展。而要想适应这个社会，总要先全面真实地了解它吧。所以，与其怨天尤人，不如反思一下自己。或许自己对这个社会的理解还不够透彻，或许很多看上去不合理的结构，并非毫无道理。 ”的确，与其郁闷不如改变~

回复
Unknown
2006年11月5日 14:32 编辑
哈哈，刚看完这本书，很有收获，窃以为是一部奇书！
不看不知道，一看吓一跳。
：）

回复
Qizhen
2006年09月5日 10:27 编辑
如果一个人一心只想做好自己的技术，其它一概不管，也应该是一个很不错选择。我也比较赞赏这样的人。但是我自己突然开始关心其管理、政治、经济等等本职技术以外的东西来了，是出于一种忧患意识。技术工作有很多种，我所从事的 IT 行业，非常不幸，是一个典型的经验贬值型行业。它的贬值速度基本等同于计算机降价的速度。我要是个医生，才不会有这种危机感。
我现在所掌握的技术，等我到 40 岁的时候，很可能就一钱不值了。当然在我到40岁之前我还有精力学习一下最前沿的知识，跟上潮流。但肯定学不过更年轻的人。但是到了50 岁怎么办呢？
我最大的担忧就是如果在 40 岁之前不转去做管理，50 岁的时候恐怕就要失业了。管理则是典型的经验增值型行业，越是老姜越值钱。干上管理，即本就不用担心失业了。

回复
Wei
2006年09月4日 18:33 编辑
有什么比驾驭人更难的工作呢？驾驭技术是我们技术人员的追求，殊不知，驾驭人这才是更重要的。通常技术人员会抱怨经理什么都不懂，但实际上，这种认识是盲目的，不信你去管理几十个人试试看。。。估计会被骂得更惨。

回复
左岸 右转
2006年08月31日 12:55 编辑
这书看起来应该是有很大的参考价值,但,心里总觉得工作永远是一件发自内心的事情,找到自己内心向往的价值并实现它好象更容易让人产生事业的激情,所以尽管这书不错,但委实提不起兴趣来读, 真的不想那么清楚……呵呵,顽固如斯,势必无药可救..

回复
永青
2006年08月31日 09:41 编辑
哈哈，这篇文章很赞！

回复
yi
2006年08月30日 23:52 编辑
看来人在社会中，还是首先要适应环境，改变自己啊，否则撞得头破血流还坚持认为自己是对的。

回复
